lib/Pipeline.js

```diff
@@ -105,7 +107,12 @@ Pipeline.processJSONWithExtras  = function(gltfWithExtras, options) {
         optimizeForVertexCache(gltfWithExtras);
     }
 
-    // run AO after optimizeForVertexCache since AO adds new attributes.
+    // run generation of tangents / binormals and AO after
+    // optimizeForVertexCache since those steps add new attributes.
+    if (defined(options.tangentsBinormals) && options.tangentsBinormals) {
```

Comment:
we have a similar function that just checks if the given variable exists (is not `undefined`), guess that's why I got a bit confused here